src/main/java/com/devonfw/ide/sonarqube/common/impl/DevonfwJavaProfile.java

```diff
@@ -112,7 +112,6 @@ private Document readQualityProfileXml() {
       logger.log(Level.WARNING, "There was a problem configuring the parser.");
       return null;
     } catch (IOException io) {
-      io.printStackTrace();
       logger.log(Level.WARNING, "There was a problem reading the file.");
       return null;
```

Comment:
fine taht `printStackTrace` was removed.
However, exceptions with their stacktrace and all causes are valuable and need to be logged with all details (see https://github.com/devonfw/devon4j/blob/develop/documentation/coding-conventions.asciidoc#catching-and-handling-exceptions). Maybe in such case it is even better to convert the `IOException` to a runtime exception (e.g. `IllegalStateException`) and let SonarQube handle it properly.

Reply:
Currently we will not be able to figure out
* which file could not be read
* why the file could not be read (e.g. permission denied, disc corrupted, remote filesystem network error, etc.)